examples/misc/01_ader_fd.ipynb

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
<p>No need for "as usual". Ref. the difference of this setup vs the one in the existing first order notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>